google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionImpl.java

```diff
@@ -208,6 +208,7 @@ static UnitOfWorkType of(TransactionMode transactionMode) {
   private AutocommitDmlMode autocommitDmlMode = AutocommitDmlMode.TRANSACTIONAL;
   private TimestampBound readOnlyStaleness = TimestampBound.strong();
   private QueryOptions queryOptions = QueryOptions.getDefaultInstance();
+  private Options.RpcPriority rpcPriority = Options.RpcPriority.HIGH;
```

Comment:
Would it be an idea to use a different default value for this field, as there is currently no way to tell the difference between a connection that has been set to use `RpcPriority.HIGH` and one that uses the default value. This means that:
1. The connection must either always explicitly pass the priority as an option to the client library.
2. OR: Rely on the fact that HIGH is the default priority, which means that if that would ever change, this code would also need to change.

Reply:
I think we want #2 here since we want default to be HIGH https://github.com/googleapis/java-spanner/pull/676/files#diff-992d480774d874d97423da09934b0395ef460130558e2fd6f7b1461ba32535c0R29
cc: @olavloite